src/user-guide/writing-suites/configuration.rst

```diff
@@ -0,0 +1,211 @@
+.. _User Guide Configuration:
+
+Workflow Configuration
+======================
+
+Cylc suites are defined in structured, validated, :cylc:conf:`flow.cylc`
+files that concisely specify the properties of, and the relationships
+between, the various tasks managed by the suite.
+
+Here we will look at the format and content of the
+:cylc:conf:`flow.cylc` file and how to configure suites.
+
+
+.. _SuiteDefinitionDirectories:
+
+Suite Configuration Directories
+-------------------------------
+
+A Cylc :term:`suite directory` contains:
```

Comment:
```suggestion
A Cylc :term:`workflow directory` contains:
```
Don't worry if this is going to spread ripples all over cylc-doc. All our codebases and brains will require a `sed 's/suites/workflows/'` at some point.